temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -207,6 +207,12 @@ def nested_child(path: Sequence[str], child: SandboxMatcher) -> SandboxMatcher:
     ``True`` and this matcher is on ``children`` of a parent.
     """
 
+    exclude: Set[str] = frozenset()  # type: ignore
```

Comment:
We can now use the usual lower-case type names for collection types.

Reply:
True, though inconsequential here I think. But would be opposed to a general porting.